server/controllers/userController.js

```diff
@@ -0,0 +1,56 @@
+const { User } = require('../models/index');
+
+const getAllUsers = async (req, res) => {
+    try { 
+        const users = await User.findAll();
+        if (users) {
+            res.status(200).json(users);
+        }
+    } catch(error) {
+        res.status(500).send(error);
+    }
+}
+
+const getUser = async (req, res) => {
+    try {
+        const userId = req.params.userId
+        const user = await User.findByPk(userId);
+        if (user) {
+            res.status(200).json(user);
+        }
+    } catch (error) {
+        res.status(500).send(error);
+    }
+}
+
+const updateUser = async (req, res) => {
+    try {
+        const newInfo = req.body;
+        const userId = req.params.userId;
+        const user = User.findByPk(userId);
```

Comment:
add a guard against when user with given userID isn't found :
(!user) res.status(404).send("error fetching user: User with ID ${userId} not found')